### ISSUE TITLE
chromaprint bugs may cause nowplaying to crash

### ISSUE THREAD

**Issue by aw-was-here** [whatsnowplaying/whats-now-playing#206]
Not sure if this bug impacts just the library or fpcalc.exe.  Anyway crash is:

```
rocessor.cpp:148: virtual void chromaprint::AudioProcessor::Consume(const int16_t*, int): Assertion `length % m_num_channels == 0' failed.
Aborted (core dumped)
```

Worse, however, is that it takes out the main process which leaves dead children.

**Comment by aw-was-here** [comment]
See https://github.com/acoustid/chromaprint/issues/90 as to the source of the crash. Until this gets fixed, it may be required to run the acoustid code in a separate process.

**Comment by aw-was-here** [comment]
fixed by #223
